src/lib/scheduler/node_queue_scheduler.hpp

```diff
@@ -12,63 +12,53 @@ namespace hyrise {
 /*
  * GENERAL SCHEDULING CONCEPT
  *
- * Everything that needs to be processed is encapsulated in tasks. For example, in the context of the database
- * the OperatorTasks encapsulates database operators (here, it only encapsulates the execute function).
- * A task will then be pushed by a Scheduler into a TaskQueue and pulled out by a Worker to be processed.
+ * Everything that needs to be processed is encapsulated in tasks. For example, in the context of the database, the
+ * OperatorTasks encapsulates database operators (here, it only encapsulates the execute function). A task will be
+ * pushed by a Scheduler into a TaskQueue and pulled out by a Worker to be processed.
  *
  *
  * TASK DEPENDENCIES
  *
- * Tasks can be dependent of each other. For example, in the context of the database, a table scan operation can be
- * dependent on a GetTable operation and so do the tasks that encapsulates these operations.
+ * Tasks can be dependent of each other. For example, a table scan operation can be dependent on a GetTable operation
+ * and so do the tasks that encapsulates these operations.
```

Comment:
`do the tasks that encapsulate these operations` or `does the task that encapsulates this operation`